cachi2/core/package_managers/yarn_classic/project.py

```diff
@@ -0,0 +1,171 @@
+"""
+Parse the relevant files of a yarn project.
+
+It also provides basic utility functions. The main logic to resolve and prefetch
```

Comment:
Does it still provide these functions?

Reply:
Yes, e.g. creating ConfigFile instances from existing files, creating Project instances from source dir

Reply:
Fair enough.